Services/CO.CDP.Organisation.WebApi/Model/Organisation.cs

```diff
@@ -28,6 +28,8 @@ public record Organisation
     public required List<PartyRole> Roles { get; init; }
 
     public required Details Details { get; init; }
+
+    public BuyerInformation? BuyerInformation { get; init; }
```

Comment:
We're not allowed to add top level elements on an OCDS-based response. Buyer information should go into the `details` element like any other extension.